ucm2/Qualcomm/x1e80100/x1e80100.conf

```diff
@@ -1,12 +1,12 @@
 Syntax 4
 
-If.LENOVOX14s {
+If.LENOVOT14s {
 	Condition {
 		Type RegexMatch
 		String "${sys:devices/virtual/dmi/id/board_vendor}-${sys:devices/virtual/dmi/id/product_family}"
```

Comment:
It may be useful to define a new variable like:

```
Define.DMI_info "${sys:devices/virtual/dmi/id/board_vendor}-${sys:devices/virtual/dmi/id/product_family}"
```

And use it in condition like `String "${var:DMI_info}"`.
```

Reply:
Okay this worked out well, too. Thanks!